include/lbann/data_readers/data_reader_jag_conduit.hpp

```diff
@@ -38,7 +38,7 @@
 #include <map>
 #include <memory>
 
-//#define _USE_IO_HANDLE_
+#define _USE_IO_HANDLE_
```

Comment:
Just FYI any symbol that starts with an underscore followed by a capital letter is reserved, per the C++ standard. I would recommend something like `DR_JAG_CONDUIT_USE_IO_HANDLE` or `LBANN_USE_IO_HANDLE`.

Reply:
good point but what does DR_ stands for?

Reply:
"DATA_READER" That's just _so_ many characters...

Reply:
I'm guessing: Data Reader???